Remove specific number of steps in update docs (#5556)
Cog-Creators/Red-DiscordBot: docs/update_red.rst

Windows
-------

> If you have Red 3.2.0 or newer, you can upgrade by following a few easy steps:

Reviewer: ```suggestion
If you have Red 3.2.0 or newer, you can upgrade by following these steps:
```

Linux & Mac
-----------

> If you have Red 3.2.0 or newer, you can upgrade by following a few easy steps:

Reviewer: ```suggestion
If you have Red 3.2.0 or newer, you can upgrade by following these steps:
```
